examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-6_Security_in_federated_compute_system/06.2_identity_security/identity_security.ipynb

```diff
@@ -0,0 +1,210 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# **Indentity Security**  \n",
+    ">note\n",
+    " This is section is the same as \n",
+    "[FLARE's Security documentation](https://nvflare.readthedocs.io/en/2.4/user_guide/security/identity_security.html)\n",
```

Comment:
if it's the same, maybe don't mention it?
otherwise, suggest: "Please read xxx for more information"

And suggest using the "main" version of the documentation link.